Snapshot build failing

Fails with

```
00:01:13.398 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-source-plugin:3.3.1:jar (attach-sources) on project microprofile-openapi-api: Presumably you have configured maven-source-plugin to execute twice in your build. You have to configure a classifier for at least one of them. -> [Help 1]
00:01:13.398 [ERROR] 
00:01:13.399 [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
00:01:13.399 [ERROR] Re-run Maven using the -X switch to enable full debug logging.
00:01:13.399 [ERROR] 
00:01:13.399 [ERROR] For more information about the errors and possible solutions, please read the following articles:
00:01:13.400 [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
00:01:13.400 [ERROR] 
00:01:13.400 [ERROR] After correcting the problems, you can resume the build with the command
00:01:13.401 [ERROR]   mvn <goals> -rf :microprofile-openapi-api
```

--- a/pom.xml
+++ b/pom.xml
@@ -19,7 +19,7 @@
     <parent>
         <groupId>org.eclipse.microprofile</groupId>
         <artifactId>microprofile-parent</artifactId>
-        <version>2.9</version>
+        <version>2.10-RC1</version>
     </parent>
 
     <groupId>org.eclipse.microprofile.openapi</groupId>

